### ISSUE TITLE
dfuse-eosio supports custom genesis account and system account prefix

### ISSUE THREAD

**Issue by cppfuns** [dfuse-io/dfuse-eosio#211]
I currently customize development based on EOSIO-eos. Compared with EOSIO-eos, the difference is the prefixes of custom genesis account and system account.

I would like to ask whether dfuse-eosio supports parameterized setting of genesis account and system account prefix? For eosio-eos doing baas, modifying the prefix of the genesis account and system account is also the most basic support.
For the existing settings, the default value can also be the current parameters of eosio-eos, and there will be no other impact

**Comment by maoueh** [comment]
Indeed, we have `eosio` hard-coded at some places throughout the code. We are open for a PR that implement the feature of making the system account configurable, it’s something good for the product, just not a priority for us.

All chains we ever need to support so far still used the `eosio` system account in some ways. It never been a problem for us for far. Note that the ecosystem itself is not sometimes well supporting other system account name. 

Turning this into a feature request and updating the title.
